### ISSUE TITLE
ConductR CLI features should discoverable (TAB completion and/or docs)

### ISSUE THREAD

**Issue by ktoso** [sbt/sbt-typesafe-conductr#45]
For example: It should be easy for a developer who issues `startBundle` to see it running without exiting sbt. This would be nicest to provide as bridge tasks like `conduct:info` which call through to the [conduct CLI](https://github.com/typesafehub/typesafe-conductr-cli) I think.

// Take this as feedback from "first time using this stuff developer", probably not high priority but nevertheless would make the offering feel complete / polished.

**Comment by huntc** [comment]
`bundleInfo` and others are there. What's missing?

**Comment by ktoso** [comment]
Ok, so it's just confusing then.

The way I (and many devs) discover tasks in sbt is via tab completion in "the interesting namespace", our tasks are not defined `... in conductr` which makes them not possible to find via:

```
[reactive-maps] conductr:<TAB>
// I'm expecting conductr tasks to be listed, currently getting nothing
```

The commands you mention are not listed in the readme, below `The following sbt-typesafe-conductr commands are available:`, I couldn't TAB-complete to them, so I didn't know about them. Once you mention the `bundle###` pattern I'm all like "oh right. sure", but it was hard to find - in a rush I didn't.

Please take this as "first time user feedback" :-)

**Comment by huntc** [comment]
Thanks Konrad! Useful feedback.

**Comment by huntc** [comment]
Fixed by #52
